scalapact-sbtplugin/src/main/scala/com/itv/scalapact/plugin/verifier/Verifier.scala

```diff
@@ -37,39 +37,47 @@ object Verifier {
 
           case \/-((b, p, c)) =>
             val address = b + "/pacts/provider/" + p + "/consumer/" + c + "/latest"
-            println(s"Attempting to fetch pact from pact broker at: $address".white.bold)
+            readPact(address)
+        }
+      }.collect {
+        case Some(s) => s
+      }
 
```

Comment:
You can reduce duplication by merging the lists in advance. If you convert `["a","b","c"] ` into `[("a", "latest"),("b", "latest"),("c", "latest")]` and add it to the versioned list then you only need the second block right?